src/components/forms/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,97 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import * as RadioGroupRadix from '@radix-ui/react-radio-group';
+import { ReactNode } from 'react';
+
+import { InputVariant } from '../styles';
+
+import { RadioOptionButton } from './RadioOptionButton';
+import { RadioOptionClassic } from './RadioOptionClassic';
+
+export interface ValueLabel {
```

Comment:
Please rename to `RadioOption`

Reply:
Do you want me to rename the file? or the folder?

Reply:
Juste the interface